mage.ts

```diff
@@ -1286,6 +1286,17 @@ declare class Mage extends NodeJS.EventEmitter {
     isCodeFileExtension(ext: string): boolean;
 
     /**
+     * The worker ID of the currently worker process
+     *
+     * Will return false if the process is the master process, or
+     * if MAGE is running in single mode.
+     *
+     * This differs from cluster.worker.id because it will remain consistent
+     * if restarted or reloaded.
+     */
+    workerId: number | false;
```

Comment:
Would `undefined` not make more sense? If we can easily avoid mixing types (I don't consider `undefined` a type), I think that's preferable.

Reply:
Agreed. I'll update.